flanksource/karina
# feat: Add support for encrypted config files

### Description

Add support for parsing encrypted config files

### Dependencies

- N/A

### Breaking Change

- [X ] No

### Testing Notes

Confirmed secure config files are correctly imported, decrypted and merged using

`karina render -c test/secure.yaml`

To replicate:
- load flanksource KMS credentials into env as per AWS/SOPS practice
- `karina render -c test/secure.yaml`
- confirm CA and ingressCA fields are correctly added to rendered output, they should match the contents of `test/minimal.yaml`

### **Links**

Closes #611
